charts/hami/templates/scheduler/webhook.yaml

```diff
@@ -20,10 +20,10 @@ webhooks:
     name: vgpu.hami.io
     namespaceSelector:
       matchExpressions:
-      - key: hami.io/webhook
+      - key: kubernetes.io/metadata.name
         operator: NotIn
         values:
-        - ignore
```

Comment:
I suggest adding a check if whitelistNamespaces is empty and keeping the existing logic

At the same time, it is recommended to keep the `hami.io/webhook: ignore` logic, and you can add one or more matchExpressions

Reply:
Done, add the empty check logic